tla-pp/src/test/scala/at/forsyte/apalache/tla/pp/TestConstSimplifier.scala

```diff
@@ -405,21 +408,74 @@ class TestConstSimplifier extends FunSuite with BeforeAndAfterEach with Checkers
     check(prop, minSuccessful(1000), sizeRange(8))
   }
 
+  test("simplifies quantifiers which rewrite to Boolean constants") {
+    val prop = forAll(gens.genType1) { t: TlaType1 =>
+      def n_x = tla.name("x") as t
+      def n_S = tla.name("S") as SetT1(t)
+      def n_p = tla.name("p") as BoolT1()
+
+      def emptySet = tla.enumSet() as SetT1(t)
+
+      // \E x \in {}: P
+      val existsEmpty = tla.exists(n_x, emptySet, n_p) as BoolT1()
+      val existsEmptyResult = simplifier.simplify(existsEmpty)
+      // \A x \in {}: P
+      val forallEmpty = tla.forall(n_x, emptySet, n_p) as BoolT1()
+      val forallEmptyResult = simplifier.simplify(forallEmpty)
+      // \E x \in S: FALSE
+      val existsFalse = tla.exists(n_x, n_S, tla.bool(false)) as BoolT1()
+      val existsFalseResult = simplifier.simplify(existsFalse)
+      // \A x \in S: TRUE
+      val forallTrue = tla.forall(n_x, n_S, tla.bool(true)) as BoolT1()
+      val forallTrueResult = simplifier.simplify(forallTrue)
+
+      existsEmptyResult shouldBe (tla.bool(false) as BoolT1()) withClue s"when simplifying ${existsEmpty.toString}"
+      forallEmptyResult shouldBe (tla.bool(true) as BoolT1()) withClue s"when simplifying ${forallEmpty.toString}"
+      forallTrueResult shouldBe (tla.bool(true) as BoolT1()) withClue s"when simplifying ${forallTrue.toString}"
+      existsFalseResult shouldBe (tla.bool(false) as BoolT1()) withClue s"when simplifying ${existsFalse.toString}"
```

Comment:
Just a small one to put the checks in the order of the definitions above.

```suggestion
      existsFalseResult shouldBe (tla.bool(false) as BoolT1()) withClue s"when simplifying ${existsFalse.toString}"
      forallTrueResult shouldBe (tla.bool(true) as BoolT1()) withClue s"when simplifying ${forallTrue.toString}"
```